### PR TITLE
Midterm

### PR DESCRIPTION
**Pull request by EvanLuke**
I don't know why for q1 the 25 input doesn't return the right number. 
Also when I run q2 and play the game it only says when I lose, not the computer. It starts going into negative numbers. I am trying to figure this out now.

### CODE REVIEW

**Comment by EvanLuke** [comment]
The program runs perfectly on my IDLE I don't know why it wont work here.

File: q2.py [hunk 1]
```diff
@@ -20,21 +20,40 @@ def playGame():
...
+        return int(sticksPicked)
+
+
 
 def subtractSticks( number ):
     global sticks 
-    
+    sticks = sticks - number
+    if sticks <=  1:
+        return True
```
Comment (halfaro1) [inline q2.py]: @EvanLuke I believe this code return `True` if there is still 1 stick remaining, which is not a losing condition for the player that left 1 stick in the pile. The loser is the next player, who has to take the 1 stick. That's the player that should return True.

**Comment by EvanLuke** [comment]
Recommitted, thanks for the help.

**Comment by halfaro1** [comment]
@EvanLuke 
Fantastic work :+1:
I'm marking this pull request as closed, and labeling it as `graded` while I finish grading these projects. When I'm done grading all the projects, and your grade should be on Blackboard, I'll change the tag to `grade updated on blackboard`

**Pull request closed without merging**